src/cocotb/simulator.pyi

```diff
@@ -0,0 +1,78 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+# generated with mypy's stubgen script
+
+from typing import Any
+
+DRIVERS: int
+ENUM: int
+GENARRAY: int
+INTEGER: int
+LOADS: int
+MEMORY: int
+MODULE: int
+NET: int
+NETARRAY: int
+OBJECTS: int
+PACKAGE: int
+REAL: int
+REG: int
+STRING: int
+STRUCTURE: int
+UNKNOWN: int
+
+class gpi_cb_hdl:
+    def deregister(self) -> None: ...
+    def __eq__(self, other: object) -> bool: ...
+    def __ne__(self, other: object) -> bool: ...
+    def __hash__(self) -> int: ...
+
+class gpi_iterator_hdl:
+    def __eq__(self, other: object) -> bool: ...
+    def __ne__(self, other: object) -> bool: ...
+    def __hash__(self) -> int: ...
+    def __iter__(self): ...
+    def __next__(self): ...
+
+class gpi_sim_hdl:
+    def get_const(self) -> bool: ...
+    def get_definition_file(self) -> str: ...
+    def get_definition_name(self) -> str: ...
+    def get_handle_by_index(self, index: int) -> gpi_sim_hdl: ...
+    def get_handle_by_name(self, name: str) -> gpi_sim_hdl: ...
+    def get_name_string(self) -> str: ...
+    def get_num_elems(self) -> int: ...
+    def get_range(self) -> tuple[int, int]: ...
+    def get_signal_val_binstr(self) -> str: ...
+    def get_signal_val_long(self) -> int: ...
+    def get_signal_val_real(self) -> float: ...
+    def get_signal_val_str(self) -> bytes: ...
+    def get_type(self) -> int: ...
+    def get_type_string(self) -> str: ...
+    def iterate(self, mode: int) -> gpi_iterator_hdl: ...
+    def set_signal_val_binstr(self, action: int, value: str) -> None: ...
+    def set_signal_val_int(self, *args, **kwargs): ...
```

Comment:
```suggestion
    def set_signal_val_int(self, action: int, value: int): ...
```
Looks like the typed definition is missing from the `PyDoc_STR()` for this method in `simulatormodule.cpp`.

Reply:
I'll fix simulatormodule.cpp as well.